tensorrt_llm/serve/scripts/benchmark_serving.py

```
@@ -736,6 +736,30 @@ async def fetch_energy_metrics(base_url: str) -> Optional[dict]:
+    Any network/parse failure is swallowed and reported as False so callers
+    can fall back to safe defaults.
+    """
+    models_url = f"{base_url}/v1/models"
+    async with aiohttp.ClientSession(trust_env=True,
+                                     timeout=AIOHTTP_TIMEOUT) as session:
+        try:
+            async with session.get(models_url) as response:
+                if response.status != 200:
+                    return False
+                payload = await response.json()
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)